cinema/models.py

```diff
@@ -134,9 +130,7 @@
         )
 
     def __str__(self):
-        return (
-            f"{str(self.movie_session)} (row: {self.row}, seat: {self.seat})"
-        )
+        return f"{str(self.movie_session)} (row: {self.row}, seat: {self.seat})"
```

Comment:
The `__str__` method for the `Ticket` model is well-implemented, providing a clear string representation of the ticket, including the movie session, row, and seat.